packages/api/src/controllers/usage.ts

```diff
@@ -385,6 +375,10 @@ app.post(
       }
     }
 
+    // New automated billing usage report
```

Comment:
Do we still need all the old reporting logic? Could be good to clean that up as well.

Reply:
We need it! the explorer is still relying on that